integr8ly/delorean
# Del 222 tag image

To verify:

1. Make sure you have a valid quy.io api token. Instructions can be found [here](https://docs.quay.io/api/), see "Generating a token" section. You can use this [application](https://quay.io/organization/integreatly/application/A2LWZS1CLEBXG8AKOSQ2?tab=gen-token) to generate a token. Once the token is created, set it via the `QUAY_TOKEN` env var.
2. Fork integreatly-operator into your org
3. Manually create a release PR against your own fork (probably against a test branch rather than master). The release branch name should be something like `release-vx.x.x`
4. Build the cli and run the following:
    ```
    ./delorean release merge-release -o <your-fork> -b <base-branch> -v x.x.x-er1
   ```
   You should see some output and the PR should be merged.
5. To tag the release image (you can test it using the integreatly-operator, just make sure use a tag that hasn't been used, and delete them afterwards from github and quay repos if they get created):
   ```
   ./delorean release tag-release -v 2.0.0-test
   ```
   This will fail if the latest image is not available on quay. You can also try wait for it:
    ```
    ./delorean release tag-release -v 2.0.0-test -w --wait-interval 1
    ```
   You should see some output and it will create the tag once the image is available on quay.
